src/device-registry/routes/v1/sites.js

```diff
@@ -481,20 +481,20 @@ router.put(
         .withMessage("distance_to_nearest_residential_road must be a number")
         .bail()
         .toFloat(),
-      body("bearing_to_kampala_center")
+      body("bearing_to_capital_city")
```

Comment:
@BenjaminSsempala 
We could perhaps also just add NEW input validations instead of "replacing" the old ones since we might not be moving at the same speed as frontend engineers. Just to allow for a gradual process to take shape.
Thanks!

Reply:
Thanks @Baalmart I've updated this